### PR TITLE
Expand `adb install` technique

### PR DESCRIPTION
**Pull request by TheDauntless**
Thank you for submitting a Pull Request to the OWASP MASTG. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

### CODE REVIEW

File: techniques/android/MASTG-TECH-0005.md [hunk 1]
```diff
@@ -3,10 +3,97 @@ title: Installing Apps
 platform: android
 ---
 
-Use `adb install` to install an APK on an emulator or connected device.
+## Basic APK installation
```
Comment (cpholguera) [inline techniques/android/MASTG-TECH-0005.md:6]: could you please fix all other title capitalizations?

https://mas.owasp.org/contributing/5_Style_Guide/#title-capitalization

```suggestion
## Basic APK Installation
```

File: techniques/android/MASTG-TECH-0005.md [hunk 2]
```diff
@@ -3,10 +3,97 @@ title: Installing Apps
...
+```bash
+adb install ./myRepackagedApp.apk
+adb: failed to install myRepackagedApp.apk: Failure [INSTALL_FAILED_UPDATE_INCOMPATIBLE: Existing package org.owasp.mas.app signatures do not match newer version; ignoring!]
+```
+
+To solve this issue, first remove the original application using `adb uninstall`:
+
+```bash
+# Uninstall based on package name
+adb uninstall org.owasp.mas.app
```
Comment (cpholguera) [inline techniques/android/MASTG-TECH-0005.md:52]: we can take the chance here to use our app:

```suggestion
adb uninstall org.owasp.mastestapp
```

**Comment by TheDauntless** [comment]
Done! Thanks for the review! 👍

**Review by cpholguera — APPROVED** [review]
Thank you!!

**Pull request merged**